### DISCUSSION TITLE
Connection handle covering the whole note and allowing to drop a connection–similar to the "Easy Connect" example

### DISCUSSION THREAD

**Discussion by ratiofu** [xyflow/xyflow#3161] [Q&A] [answered]
I am trying to build an interaction similar to the [Easy Connect example](https://reactflow.dev/docs/examples/nodes/easy-connect/), with a crucial enhancement: in the current public example, an edge has to be dragged to the center of the node before it is recognized as a connection. How would I modify this example to have the entire handle area count as "valid" for making a connection?

I feel like I am missing something very simple here. I've read the documentation and API several times and looked at several Q&A discussions, but can't figure it out.

**Comment by moklick** [comment]
This seems like a bug. If a handle is below the mouse pointer we should use the handle, even if it's not inside the connection radius. As a workaround you can increase the `connectionRadius` prop https://reactflow.dev/docs/api/react-flow-props/#connectionradius 

I'll let you know when this is fixed.

**Reply by moklick** [reply, marked as answer]
we fixed that behaviour in v11.7.4 and updated the example: https://reactflow.dev/docs/examples/nodes/easy-connect/ 

Does it work for you @ratiofu ?

**Reply by ratiofu** [reply]
Thanks @moklick. The example now behaves as I would expect. I'll update on my end shortly and confirm.

**Reply by moklick** [reply]
It's a bit of a hack since it not supported out of the box..

**Reply by d-reinhold** [reply]
@moklick This still seems to be an issue on the example site, as well as in my application. What is the "hack" you're referring to?

https://github.com/xyflow/xyflow/assets/815285/687da012-5c32-41d8-b46c-6c000f3827ff

**Reply by hitoshisatow** [reply]
I am also having trouble with this and have been trying to find a solution for the last day. It seems like the issue still persists. Is the only workaround to increase the connectionRadius?
